src/Microsoft.VisualStudio.SDK.Analyzers.Tests/VSSDK006CheckServicesExistAnalyzerTests.cs

```diff
@@ -572,12 +557,12 @@ class Test : Package {
 }
 ";
 
-        this.VerifyCSharpDiagnostic(test, this.CreateDiagnostic(8, 35, 15));
-        this.VerifyNoCSharpFixOffered(test);
+        var expected = this.CreateDiagnostic(8, 35, 15);
+        await Verify.VerifyCodeFixAsync(test, expected, test);
```

Comment:
@sharwell Why did you use `VerifyCodeFixAsync(test, expected, test);` here instead of `VerifyAnalyzerAsync(test, expected);`?

There is no code fix offered for this, and I'm surprised then that we're calling a method to verify the code fix. Also, I'm surprised it succeeds. I would expect a test framework to throw "no code fix offered" in a case like this.

Reply:
OK, fair enough. I guess that way, if we enhance the code fix provider to offer one, it causes test failures, giving us an opportunity to add an assertion about what the fix should be, rather than quietly continuing without testing it.
Thanks.